test/4-stow-permissions-test.js

```diff
@@ -51,12 +59,44 @@ describe('Stow-permissions', async () => {
       assert.equal(perm.dataUri, testSharedUri);
     });
     it(
-      'should return the permission info if viewer does not have access',
+      'should not return the permission info if viewer does not have access',
       async () => {
         const perm = await stow.getPermission(testDataHash, user3);
         assert.isFalse(perm.canAccess);
         assert.isEmpty(perm.dataUri);
       },
     );
   });
+  describe('grant permission', () => {
+    beforeEach('setup ipfs record', async function () {
+      this.timeout(10000);
```

Comment:
Is this necessary?  If so please add a comment.

Reply:
Yes because of ipfs loading the test times out at 2000ms but this can be removed once we switch to new scheme